Source/Csla.Blazor/BusinessRulesValidationMessages.razor

```diff
@@ -0,0 +1,21 @@
+﻿@namespace Csla.Blazor
+@inherits BusinessRulesValidationMessageBase
+
+<div class="@WrapperClass" aria-live="polite">
```

Comment:
I wonder if we don't need 3 wrappers for more control over how the info/warn/error text is displayed?

Reply:
Yes, I did wonder that. I thought that the outer wrapper is important in getting the messages to all line up with one another, and putting another wrapper within it by type might be overkill.

Were you thinking instead of the current one, or in addition?

Reply:
I don't know the answer for sure - I'm not an html expert :)

But it seems like a UI might want to arrange info/warn/error in different ways (side-by-side instead of vertical, etc.) maybe?

So maybe there needs to be an outer-most wrapper, and then a wrapper for each type of message?

Reply:
I'm intending to discuss this in more detail with our UI expert tomorrow. I think it has merit, but I'd value his input - he might be better at HTML than either of us, and is exactly the sort of person we are targeting with the change.

I started the conversations with him about the flexibility and accessibility of this component on Monday, but he's not in the office on even days, and needed some time to mull it over anyway. 

I'm hoping to complete the PR for this component before the drop on Monday, but it's sensible to leave it out if we have any doubts at all, as once it's there, it can't really change without breaking people.

Reply:
UI expert said he could imagine no need for this, but at the same time saw no reason _not_ to, so for the sake of flexibility, I'll make this change. Tick! Thanks.